tests/sys/psa_crypto_mac/test_hmac_sha384.c

```diff
@@ -0,0 +1,279 @@
+/*
+ * Copyright (C) 2024 TU Dresden
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_psa_crypto
+ * @defgroup    sys_psa_crypto_hmac  PSA Generic HMAC
+ *
+ *
+ * @file        tests-hashes-sha384-hmac.c
+ * @brief       Unit tests for validating generic PSA Crypto HMAC implementation using SHA-384.
+ *
+ * @author      Daria Zatokovenko <daria.zatokovenko@mailbox.tu-dresden.de>
+ *
+ */
+
+#include <limits.h>
+#include <string.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "psa/crypto.h"
+
+#include "embUnit.h"
+
+#include "hashes/sha384.h"
```

Comment:
```suggestion
```

The PSA API test shouldn't depend on the `hashes` module.

Reply:
I agree, i will rework the tests to use the official PSA hashing API instead.